libs/sdk-backend-tiger/src/auth.ts

```diff
@@ -163,3 +178,35 @@ export function createTigerAuthenticationUrl(
         authenticationFlow.returnRedirectParam
     }=${encodeURIComponent(returnAddress)}`;
 }
+
+/**
+ * Given authentication context and the authentication error, this implementation of `NotAuthenticatedHandler`
+ * will redirect current window to location where Tiger authentication flow will start.
+ *
+ * The location will be setup with correct return address so that when the flow finishes successfully, the
+ * browser window will be redirected from whence it came.
+ *
+ * @remarks See also {@link createTigerAuthenticationUrl}; this function is used to construct the URL. You may use
+ *  it when build your own handler.
+ * @param context - authentication context
+ * @param error - not authenticated error, must contain the `authenticationFlow` information otherwise the
+ *  handler just logs an error and does nothing
+ * @public
+ */
+export function redirectToTigerAuthentication(
+    context: IAuthenticationContext,
+    error: NotAuthenticated,
+): void {
+    if (!error.authenticationFlow) {
+        // eslint-disable-next-line no-console
+        console.error("Analytical Backend did not provide detail where to start authentication flow. ");
```

Comment:
Should this throw an error? Or how would the code recover from this?

Reply:
I considered invariant here. Then thought its perhaps too aggressive for a 'sensitive' area such as the callback that may be invoked for just about any call to analytical backend service. The invariant error would fly from here, obscuring the NotAuthenticated. The invariant will be eventually wrapped in UnexpectedError and sent out through onError callbacks.

i have no strong feelings about the current approach other that i find it 'smoother' than the invariant.

Reply:
Hm, good point. I never am sure if in "sensitive" areas we should be more or less trigger-happy. But since the error would make things more hidden, maybe your solution is better as is.